### DISCUSSION TITLE
Community Call Fri Sep 15 at 12noon Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#436] [Announcements]
Agenda

- Comments on pi-Base-related preprint: https://github.com/orgs/pi-base/discussions/432 (talk this afternoon in Carolinas seminar)
- Possible funding mechanism: https://new.nsf.gov/funding/opportunities/cyberinfrastructure-sustained-scientific (due Dec)
- Converses of theoerems, see issue at https://github.com/pi-base/web/issues/72

I'm making a new permalink for these Zoom calls: https://southalabama.zoom.us/j/93833246214

**Comment by StevenClontz** [comment]
Reminder of today's meeting in 11 minutes. I expect this to be a quicker checkin given the sparse agenda, unless anyone has anything to add.

**Comment by prabau** [comment]
To illustrate what we discussed about converses.  The converse of the enhanced Nagata-Smirnov:
https://topology.pi-base.org/theorems/T000023 (regular + sigma-locally finite base ==> pdeudometrizable)
holds, but we did not add it to pi-base.

The converse requires showing [pseudometrizable ==> regular].  As you can see below, it requires a long list of theorems:
https://topology.pi-base.org/spaces?q=pseudometrizable%2B~regular

**Comment by StevenClontz** [comment]
Note: This new AMS publication https://www.ams.org/publications/journals/journalsframework/cams is diamond open access, which feels like a good fit for an expository paper on our open-source project.

**Comment by prabau** [comment]
Wikipedia's Project Mathematics' take on the use of mathse and mo:
https://en.wikipedia.org/wiki/Wikipedia_talk:WikiProject_Mathematics/Archive/2023/Jun#On_the_use_of_Mathematics_StackExchange_and_MathOverflow_as_references

**Reply by StevenClontz** [reply]
I want to poo-poo this reluctance to accept MathOverflow citations (I'm more sympathetic to Math.SE hesitation as it doesn't get as much scrutiny, perhaps). But then again it wasn't all that long ago that I thought that pi-Base should similarly be restricted to content reflected in peer-reviewed journals/books. 🤷
